src/core/brand/SvgGuardianLiveLogo.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+	SvgGuardianLiveLogo,
+	SvgGuardianLiveLogoProps,
+} from './SvgGuardianLiveLogo';
+
+export default {
+	title: 'Source/src-brand/SvgGuardianLiveLogo',
+	component: SvgGuardianLiveLogo,
+	argTypes: {
+		width: {
+			control: { type: 'range', min: 10, max: 600 },
+		},
+	},
+};
+
+export const Demo = (args: SvgGuardianLiveLogoProps) => (
+	<SvgGuardianLiveLogo {...args} />
+);
+Demo.storyName = 'SvgGuardianLiveLogo';
```

Comment:
Would it be better just to name the story `SvgGuardianLiveLogo` rather than overriding?

```suggestion
export const SvgGuardianLiveLogo = (args: SvgGuardianLiveLogoProps) => (
	<SvgGuardianLiveLogo {...args} />
);
```

Reply:
Also, should we make all of the `Demo` stories just be the name of the component?

Reply:
i did this here because these stories don't have multiple states, so storybook hoists the single story to the top level. wihtout the rename you either have to import `SvgGuardianLiveLogo` `as` something else so you can give the story the right name (`SvgGuardianLiveLogo`) or the nav shows `Demo` as the name, rather than expose the export name